### PR TITLE
Drop 1.11 jobs and create 1.15 jobs

### PR DESCRIPTION
**Pull request by Katharine**
This PR drops 1.11 jobs and creates 1.15 jobs by:

* `bazel run //experiment:prepare_release_branch`
* Manually updating the two things tagged `manual-release-bump-required`
* Fiddling with the testgrid config a bit (there is still work to do here)
* Updating the README

I took the opinion that if it's not in a release-blocking or a release-informing dashboard, it is not my problem if it becomes stale.

/cc @spiffxp

### CODE REVIEW

**Comment by spiffxp** [comment]
/lgtm
this matches what I saw when I ran this locally, and spot checks ok

/hold
for review by @imkin

File: testgrid/config.yaml [hunk 1]
```diff
@@ -2617,10 +2617,18 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-reboot
     description: "Uses kubetest to run a subset of e2e tests (+Feature:Reboot) against a cluster created with cluster/kube-up.sh"
 
+- name: sig-release-1.15-blocking
+- name: sig-release-1.15-informing
+- name: sig-release-1.15-all
```
Comment (imkin) [inline testgrid/config.yaml]: We are not going to be having `-all` for 1.15 I believe. We are transitioning to `-informing`. As far as I remember we decided to do this incrementally (fork from master-blocking and master-informing into the new releases and ultimately `-all` dashbaords will retire once 1.14 release expires).

Comment (imkin) [reply testgrid/config.yaml]: We could remove `-all` dashboard in the coming weeks incrementally.

Comment (spiffxp) [reply testgrid/config.yaml]: ref: https://github.com/kubernetes/test-infra/issues/11977#issuecomment-492450273 for dropping -all

**Pull request merged**